src/main/java/friday/logic/commands/FindCommand.java

```diff
@@ -35,6 +37,7 @@ public CommandResult execute(Model model) {
 
     @Override
     public boolean equals(Object other) {
+        assert true;
```

Comment:
Similar to the comment above, I believe we should try to use `assert` more productively 🙂